runtime/cudaq/cudaq.cpp

```diff
@@ -185,6 +185,11 @@ void broadcast(std::vector<double> &data, int rootRank) {
   commPlugin->broadcast(data, rootRank);
 }
 
+void broadcast(std::string &data, int rootRank) {
```

Comment:
```suggestion
void broadcast(const std::string &data, int rootRank) {
```
?

Reply:
The semantics of these `broadcast` APIs is that other ranks (not `rootRank`) will have its `data` updated; hence cannot be `const`.

Reply:
+1